### PR TITLE
refactor db examples

### PR DESCRIPTION
**Pull request by volosatik**
move import.sh from exmples to templates
make dependecy config-instance-database as everywhere 
change network from app to db cause of naming purpose
partly closes VKPRGM-51
closes VKPRGM-35

### CODE REVIEW

File: examples/db/database/main.tf [hunk 1]
```diff
@@ -1,30 +1,6 @@
...
-
-  depends_on = [
-    vkcs_networking_router_interface.db
-  ]
-}
-
 resource "vkcs_db_database" "db-database" {
   name        = "testdb"
-  dbms_id     = vkcs_db_instance.db-instance.id
+  dbms_id     = vkcs_db_instance.basic.id
```
Comment (ftersin) [inline examples/db/database/main.tf:3]: lets name it basic_ms. otherwise charset/collate may confuse someone

File: examples/db/instance/main.tf [hunk 2]
```diff
@@ -5,12 +5,13 @@ resource "vkcs_db_instance" "basic" {
   flavor_id         = data.vkcs_compute_flavor.basic.id
   datastore {
     type    = "mysql"
-    version = "5.7"
+    version = "8.0"
   }
+  configuration_id = vkcs_db_config_group.db-config-group.id
```
Comment (ftersin) [inline examples/db/instance/main.tf:10]: lets place it before capabilities. and name it mysql

File: examples/db/database/main.tf (hunk 1 above)
Comment (ftersin) [reply examples/db/database/main.tf:3]: not basic - we use there capabilities and configgroup, so mysql

Comment (volosatik) [reply examples/db/database/main.tf]: yes i renamed to mysql and made all dependencies like mysql-db mysql-user etc

File: examples/db/instance/main.tf (hunk 2 above)
Comment (volosatik) [reply examples/db/instance/main.tf]: i named it mysql-80 ( as i checked if config is verA and in instance datastore has verB there is an error)

**Review by ftersin — APPROVED** [review]

**Pull request merged**